lib/kuby/environment.rb

```diff
@@ -4,19 +4,19 @@ class Environment
     attr_reader :name, :definition
     attr_accessor :configured
 
-    alias_method :configured?, :configured
+    alias configured? configured
```

Comment:
Let's keep `alias_method`, it's a lot more flexible than `alias`, can be overridden, etc.